### PR TITLE
🖍Clean up amp-pinterest button styling

### PR DESCRIPTION
**Pull request by caroqliu**
- Make a style change from #20706 to be more compatible with existing uses.

Occasionally, customized CSS may overwrite rectangular button classes with a developer-inserted circular Pinterest logo. This change aids those use instances by overriding rectangular button styling with round button styling when the width and height of the inserted amp-pinterest element are 1:1.

### CODE REVIEW

**Comment by torch2424** [comment]
Triaging to @cvializ , feel free to re-assign 😄

File: extensions/amp-pinterest/0.1/save-button.js [hunk 1]
```diff
@@ -53,7 +53,8 @@ export class SaveButton {
     this.color = rootElement.getAttribute('data-color');
     this.count = rootElement.getAttribute('data-count');
     this.lang = rootElement.getAttribute('data-lang');
-    this.round = rootElement.getAttribute('data-round');
+    this.round = rootElement.getAttribute('data-round') ||
+        rootElement.getAttribute('height') == rootElement.getAttribute('width');
```
Comment (gmajoulet) [inline extensions/amp-pinterest/0.1/save-button.js]: Are the `height` and `width` attributes required? If they're not provided by the publisher it'd come down to `undefined === undefined`, is this expected?

Also, not a big deal, but I think the internal JavaScript styleguide asks for strict equality checks when possible :)

Comment (cvializ) [reply extensions/amp-pinterest/0.1/save-button.js]: +1, amp-pinterest supports layouts supported by `isLayoutSizeDefined`, so if the publisher configured `<amp-pinterest layout="fixed-height" ...>` then the `width` attribute could be null

**Comment by cvializ** [comment]
Thanks @caroqliu! It looks like this fixed all the wonky cases I saw on cookbook.

File: extensions/amp-pinterest/0.1/save-button.js (hunk 1 above)
Comment (caroqliu) [reply extensions/amp-pinterest/0.1/save-button.js]: Good catch. I added checks to hasAttribute() on both attributes before assessing equality.

**Pull request merged**